### ISSUE TITLE
empty user agent in curl header does not work

### ISSUE THREAD

**Issue by dkwo** [lobsters/lobsters#1168]
`curl -H "User-Agent:" "https://lobste.rs"` returns empty, while adding anything after `user agent` works. is this an intentional feature?

**Comment by pushcx** [comment]
Nope. This bug is probably caused by `rack_attack.rb` assuming there'd be a string.
